docs/install/cluster.md

```diff
@@ -0,0 +1,18 @@
+# Start and connect to master
+
+## Start an EMR Cluster (Optional)
```

Comment:
probably want to add a few words why EMR is optional any alternatives, etc.

Reply:
Removed optional